include/miopen/miopen.h

```diff
@@ -5624,14 +5624,14 @@ typedef enum
  * @param handle                   MIOpen Handle (input)
  * @param xDesc                    Tensor descriptor for data input tensor x (input)
  * @param dim                      Dimensions to sum. (input)
- * @param yDesc                    Tensor descriptor for output data tensor y (input)
+ * @param reduceDesc               Tensor descriptor for output data tensor y (input)
```

Comment:
Similar libraries use the concept of "reduce descriptor" to hold a reduce-specific information, e.g. reduce op, datatype, index(es) to reduce, etc. Therefore, I recommend reverting the renaming of yDesc to reduceDesc everywhere in the public API.

Reply:
[resolved]